### PR TITLE
[E_CLASSROOM-169][BE] Functionality for Friends Activities

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-169

### Definition of Done
- [x] Retrieve most recent activities from followed user accounts.
- [x] Retrieved data should be limited to 10.

#### Main note
- 
#### Pages Affected
- http://localhost:82/api/v1/followedLog

#### Scenarios/ Test Cases
- [x] Retrieve most recent activities from followed user accounts.
- [x] Retrieved data should be limited to 10.` 

### Screenshots
followedActivities

### CODE REVIEW

**Comment by Johnpaullim** [comment]
### Working 

- [x] Able to Retrieve most recent activities from followed user accounts.
- [x] Able to displayed and  Retrieved data  be limited to 10
- [x] Functionality for friends activities is working


<details><summary>Screenshot</summary>

https://user-images.githubusercontent.com/89845252/145328752-74398a96-c4fa-42ea-8694-db625d8d1d4b.mp4

</details> 

- Total Test time: 0.25 hr

File: app/Http/Controllers/API/v1/Student/StudentActivitiesController.php [hunk 1]
```diff
@@ -6,6 +6,8 @@
 use Illuminate\Http\Request;
 use App\Models\User;
 use Spatie\Activitylog\Models\Activity;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Auth;
```
Comment (RunstyleX) [inline app/Http/Controllers/API/v1/Student/StudentActivitiesController.php]:
```suggestion
use Illuminate\Support\Facades\Auth;
use Illuminate\Support\Facades\DB;
```
Please arrange alphabetically

**Comment by RunstyleX** [comment]
Please fix PR Title to `[BE] Functionality for Friends Activities`

Please fix commits

File: app/Http/Controllers/API/v1/Student/StudentActivitiesController.php [hunk 2]
```diff
@@ -21,4 +24,20 @@ public function studentActivities(Request $request)
        return $this->successResponse(['data' => $activities], 200);
 
     }
+
+    public function followedActivities(Request $request)
+    {
+        $followedUsers = DB::table('user_follower')
```
Comment (RunstyleX) [inline app/Http/Controllers/API/v1/Student/StudentActivitiesController.php:30]: Please use the UserFollower Model and refrain from using db:table unless needed..

You can refer to the function above on how to access it.

**Pull request merged**